docs/writing-docs/tutorials.md

```diff
@@ -134,9 +145,40 @@ The title is on the first line and uses a _level 1_ heading, like:
 # Light blaster
 ```
 
+### Activities (V2)
+
+Tutorials with version **2** specified in the metadata table have multiple _activities_, each consisting of several steps. An activity begins with a _level 2_ heading (``##``) followed by the activity name, which will be displayed in a dropdown next to The activity can have any number of steps, but no activity-specific text.
```

Comment:
I think the “multi level activity” should be a property not a absolute version number. What happens when we come up with tutorials v3?

Reply:
What do you mean by property? Like, we infer anything with one heading less than the "step" as the "activity" divider? The concept of an "activity" in the code will transfer easily between versions, the only difference would be the regex for parsing